packages/create-svelte/index.js

```diff
@@ -3,7 +3,7 @@ import path from 'node:path';
 import { mkdirp, copy, dist } from './utils.js';
 
 /** @type {import('./types/index.js').create} */
-export async function create(cwd, options) {
+export function create(cwd, options) {
```

Comment:
This is a breaking change, this API is exposed publicly. And making it non-async blocks us from using async stuff in there in the future. Why this change?